include/vcpkg/base/messages.h

```diff
@@ -484,19 +488,18 @@ namespace vcpkg
                     "ignoring --{option} since a port name argument was provided");
     DECLARE_MESSAGE(AddVersionLoadPortFailed, (msg::package_name), "", "can't load port {package_name}");
     DECLARE_MESSAGE(AddVersionNewFile, (), "", "(new file)");
-    DECLARE_MESSAGE(AddVersionNewShaIs, (msg::value), "{value} is a 40-digit hexadecimal SHA", "new SHA: {value}");
+    DECLARE_MESSAGE(AddVersionNewShaIs, (msg::commit_sha), "", "new SHA: {commit_sha}");
     DECLARE_MESSAGE(AddVersionNoFilesUpdated, (), "", "No files were updated");
     DECLARE_MESSAGE(AddVersionNoFilesUpdatedForPort,
                     (msg::package_name),
                     "",
                     "No files were updated for {package_name}");
     DECLARE_MESSAGE(AddVersionNoGitSha, (msg::package_name), "", "can't obtain SHA for port {package_name}");
-    DECLARE_MESSAGE(AddVersionOldShaIs, (msg::value), "{value} is a 40-digit hexadecimal SHA", "old SHA: {value}");
+    DECLARE_MESSAGE(AddVersionOldShaIs, (msg::commit_sha), "", "old SHA: {commit_sha}");
     DECLARE_MESSAGE(AddVersionOverwriteOptionSuggestion,
                     (msg::option),
                     "The -- before {option} must be preserved as they're part of the help message for the user.",
```

Comment:
```suggestion
                    "The -- before {option} must be preserved as their part of the help message for the user.",
```

I know it's not your edit, but I think this should be a possessive pronoun instead. Please ignore if I am wrong :)

Reply:
The contraction form is intended here. The `--`s don't own the help message, they are a part of the help message. I'll add the missing 'a'